src/zql/integration.test.ts

```diff
@@ -622,3 +622,50 @@ test('or where', async () => {
 
   await r.close();
 });
+
+test('not', async () => {
+  const {q, r} = setup();
+  const issues: Issue[] = [
+    {
+      id: 'a',
+      title: 'foo',
+      status: 'open',
+      priority: 'high',
+      assignee: 'charles',
+      created: Date.now(),
+      updated: Date.now(),
+    },
+    {
+      id: 'b',
+      title: 'bar',
+      status: 'open',
+      priority: 'medium',
+      assignee: 'bob',
+      created: Date.now(),
+      updated: Date.now(),
+    },
+    {
+      id: 'c',
+      title: 'baz',
+      status: 'closed',
+      priority: 'low',
+      assignee: 'alice',
+      created: Date.now(),
+      updated: Date.now(),
+    },
+  ] as const;
+  await Promise.all(issues.map(r.mutate.initIssue));
+
+  const stmt = q
+    .select('id')
+    .where(not(expression('status', '=', 'closed')))
```

Comment:
I'm not a fan of the asymmetry.  `not` takes an expression but `where` without `not` doesn't?

I prefer `expr` over `expression` as well if we keep the function.

Reply:
I think we want to expand on what we support but I didn't want to deal with the API design in this PR.

I think we should spend some time on the API because now it is not beautiful